resources/Datastore.py

```diff
@@ -19,4 +19,4 @@ def get_type(self):
         if "local" in self.name:
             return "local"
         else:
-            return "None"
+            return "Unknown"
```

Comment:
What is this about?

Reply:
This is not related to this topic. But this annoys me since weeks. Datastore type `Unknown`  is much better understandable in this sense.